utility/firebase.js

```diff
@@ -211,6 +212,70 @@ export const deleteFaq = async (faqID) => {
   await db.collection(faqCollection).doc(faqID).delete();
 };
 
+export const updateSponsor = async (website, sponsor) => {
+  if (sponsor.id) {
+    const ref = db
+      .collection('Hackathons')
+      .doc(website)
+      .collection('Sponsors')
+      .doc(sponsor.id);
+    await ref.set(sponsor);
+    return sponsor.id;
+  }
+  delete sponsor.id;
+  const ref = db
+    .collection('Hackathons')
+    .doc(website)
+    .collection('Sponsors')
+    .doc();
+  await ref.set(sponsor);
+  return ref.id;
+};
+
+export const deleteSponsor = async (website, sponsorId) => {
+  const ref = db
+    .collection('Hackathons')
+    .doc(website)
```

Comment:
Hmm getting this when I load your branch locally. 
![Screen Shot 2020-09-11 at 11 45 17 AM](https://user-images.githubusercontent.com/38872354/92961825-cc313b80-f424-11ea-84d5-1781c1321cdb.jpg)